packages/shared/tests/routes/normalizeRoutePath.spec.ts

```diff
@@ -204,42 +204,3 @@ describe('should normalize clean paths correctly', () => {
     }),
   )
 })
-
-describe('should normalize paths with query correctly', () => {
```

Comment:
So we are not keeping hash & queries anymore in `normalizeRoutePath`. Any potential breakings?

Also, per [this comment](https://github.com/vuepress/core/pull/1562#issuecomment-2119662222):

> All parameters which is expected with no hash and queries are renamed as pathname

Should this method be renamed to `normalizeRoutePathname`?

Reply:
Nope,  routePath is for route.path, and routeFullPath is for route.fullPath.  The former does not have any hash or queries on it.

For the second one, the answer might be yes,  But before the rc9,  That function does not support harsh or queries,  So it is yes to say that we introduce a potential breaking changes in this pr, but it is likely not to break any functions downstream.

Reply:
I understand that for url definition,  This world should be named as pathname,  But we are all converting them to route  related things and route path and route full path are the things that they should be compared or set.

So I decided to name them as the things I described earlier